<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArgFileForm</class>
 <widget class="QWidget" name="ArgFileForm">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Open File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemoveSelected">
       <property name="text">
        <string>Remove Selected</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>75</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ArgFileForm</receiver>
   <slot>fileButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRemoveSelected</sender>
   <signal>clicked()</signal>
   <receiver>ArgFileForm</receiver>
   <slot>RemoveSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileButtonPressed()</slot>
  <slot>RemoveSelected()</slot>
 </slots>
</ui>
